.github/workflows/ci.yaml

```
@@ -707,6 +707,24 @@ jobs:
       - run: cargo binstall --no-confirm cargo-shear
       - run: cargo shear
 
+  ty-completion-evaluation:
+    name: "ty completion evaluation"
+    runs-on: depot-ubuntu-22.04-16
+    needs: determine_changes
+    if: ${{ needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main' }}
```

Should we limit this task to only run on ty changes
```suggestion
    if: ${{ needs.determine_changes.outputs.ty == 'true' || github.ref == 'refs/heads/main' }}
```